Products/CMFPlone/profiles/default/metadata.xml

```diff
@@ -1,4 +1,4 @@
 <?xml version="1.0"?>
 <metadata>
-  <version>5019</version>
+  <version>5020</version>
```

Comment:
@b4oshany This broke the tests. CMFPlone is an exception. You should not update its profile version by your self. It's updated in sync with steps at plone.app.upgrade.